inductiva/_cli/storage/remove.py

```diff
@@ -0,0 +1,29 @@
+"""Remove the user's remote storage contents via CLI."""
+
+from inductiva import storage
+
+
+def remove(args):
+    """Remove user's remote storage contents."""
+    path = args.path
+    confirm = args.confirm
+
+    if not confirm:
+        prompt = input(f"Are you sure you want to remove {path}? (y/n)")
+        confirm = prompt.lower() in ["y", "ye", "yes"]
+
+    if confirm:
+        print("Removing %s in the remote storage", path)
+        storage.rmdir(path)
```

Comment:
Don't we have to set `confirm=True` in the rmdir call?:

```suggestion
        storage.rmdir(path, confirm=True)
```